cli/src/base/AppCommandBase.ts

```diff
@@ -0,0 +1,54 @@
+import { IAppMetadata } from '@joystream/metadata-protobuf'
+import MembershipsCommandBase from './MembershipsCommandBase'
+import { Bytes } from '@polkadot/types/primitive'
+import { u64 } from '@polkadot/types'
+
+export default abstract class AppCommandBase extends MembershipsCommandBase {
+  appMetadataForm = {
+    description: 'App description?',
+    authKey: 'App public auth key?',
+    websiteUrl: 'Site URL where user can learn more about the app?',
+    useUri: 'App URL?',
+    platforms: 'Platforms supported by app? Format eg.: "web,mobile,native"',
+    category: 'Categories covered by app? Format eg.: "messaging,adult"',
```

Comment:
Only one category per app is currently supported, so I think the description should be updated
```suggestion
    category: 'Category of the app? Format eg.: "messaging"',
```